frontend/src/redux/api/ticketApi.js

```diff
@@ -24,8 +24,14 @@ export const getTickets = async () => {
   }
 };
 
-export const getSubTickets = async () => {
+export const getSubTickets = async (payload) => {
```

Comment:
I think we'll want to keep the existing getSubTicket call and make a new one for specific ticketIDs

Reply:
added